eprihlaska/templates/application_form.html

```diff
@@ -345,6 +345,16 @@ <h3>bakalárske štúdium - prvý stupeň</h3>
     </div>
   </div>
 
+{% if session['note'] %}
```

Comment:
@tvinar Toto zabezpeci, ze sa v PDF/tlacenej prihlaske "Poznamka" objavi az ked nebude prazdna. Ak to nieje cielom, treba dat vediet, vybavime to inak.

Vdaka!

Reply:
informatika, bioinformatika
(aplikovaná informatika ani učiteľstvá nie, pretože tie nezohľadňujú známky na koncoročných vysvedčeniach)

Reply:
@tvinar je to sice odpoved na inu otazku, ale aj tak dakujem! :)